http-server-netty/src/test/groovy/io/micronaut/http/server/netty/binding/PaginationRequest.java

```diff
@@ -0,0 +1,6 @@
+package io.micronaut.http.server.netty.binding;
+
+import io.micronaut.core.annotation.Introspected;
+
+@Introspected
+public record PaginationRequest(Integer page, Integer size) {}
```

Comment:
@MarianConstantinMarica just rename `PaginationRequest.java` to `PaginationRequest.groovy` and after this all must work correctly